themes/nekolight/README.md

```diff
@@ -0,0 +1,16 @@
+# NekoLight prompt theme
+
+Nekolight is a starship-based oh my bash prompt theme designed to be simple and lightweight.
+
+## 🚀 Installation
+
+Prerequisites
+- A Nerd Font installed and enabled in your terminal (for example, try the FiraCode Nerd Font).
```

Comment:
I'd suggest including also a Powerline Font as an option. U+E0A0 as a branch icon is available in both Nerd Fonts and Powerline Fonts.

```suggestion
- A Nerd Font or Powerline Font installed and enabled in your terminal (for example, try the FiraCode Nerd Font).
```